### PR TITLE
doc: make the chart installation doc easier to understand.

### PR DESCRIPTION
**Pull request by jwcesign**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Paragraph segmentation blurs, making it clearer.

**Which issue(s) this PR fixes**:
Fixes #none

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: charts/karmada/README.md [hunk 1]
```diff
@@ -23,41 +23,41 @@ To install the chart with the release name `karmada` in namespace `karmada-syste
 
 - local installation
```
Comment (RainbowMango) [inline charts/karmada/README.md:24]: Markdown manages chapters by [Heading](https://www.markdownguide.org/basic-syntax/#headings), why not give the `local installation` and `remote installation` heading level 3, i mean `### Local installation`.

**Review by RainbowMango — APPROVED** [review]
/approve
Thanks.

**Review by chaunceyjiang — COMMENTED** [review]
/lgtm

**Pull request merged**